spring-cloud-app-broker-deployer-cloudfoundry/src/main/java/org/springframework/cloud/appbroker/deployer/cloudfoundry/CloudFoundryAppDeployer.java

```diff
@@ -547,10 +606,13 @@ private String getLastUpdatedPackageId(List<PackageResource> packageResources) {
 					.build())
 				.type(PackageType.BITS)
 				.build())
-			.doOnRequest(l -> LOG.debug("Creating package for application {}", applicationId))
-			.doOnSuccess(response -> LOG.info("Created package for application {}", applicationId))
-			.doOnError(e -> LOG
-				.warn(String.format("Error creating package for application %s: %s", applicationId, e.getMessage())));
+			.doOnRequest(l -> LOG.debug("Creating package. applicationId={}", applicationId))
+			.doOnSuccess(response -> {
+				LOG.info("Success creating package. applicationId={}", applicationId);
+				LOG.debug(RESPONSE_LOG_TEMPLATE, response);
+			})
+			.doOnError(e -> LOG.warn(String.format("Error creating package. applicationId=%s, " + ERROR_LOG_TEMPLATE,
```

Comment:
Should this be logged at error level?

Reply:
When I was reviewing `CloudFoundryAppDeployer`, I didn't change any of the existing levels. There are many warns in there, where we use error everywhere else. I like standardizing on error, so I'm glad you caught that.

Reply:
Changed all of the warns to errors